mephisto/core/task_launcher.py

```diff
@@ -68,9 +68,12 @@ def __init__(
         self.unlaunched_units: Dict[str, Unit] = {}
         self.keep_launching_units: bool = False
         self.finished_generators: bool = False
+        self.assignment_thread_done: bool = True
+
         self.unlaunched_units_access_condition = threading.Condition()
         if isinstance(self.assignment_data_iterable, types.GeneratorType):
             self.generator_type = GeneratorType.ASSIGNMENT
+            self.assignments_thread = False
```

Comment:
What is this added for?

Reply:
it is a typo, it should be `self.assignment_thread_done` flag.